Modules/@babylonjs/react-native/shared/BabylonNative.cpp

```diff
@@ -14,12 +14,18 @@ namespace Babylon
 {
     using namespace facebook;
 
+    namespace
+    {
+        Dispatcher g_inlineDispatcher{[](const std::function<void()>& func) { func(); }};
```

Comment:
>g_inlineDispatcher [](start = 19, length = 18)

is the thought here to have synchronous execution on the calling thread?

Reply:
Yes that's exactly right. It just makes it a little easier to write the conditional logic in `UpdateView`.